plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -3,17 +3,22 @@
  */
 import { __ } from '@wordpress/i18n';
 import { createInterpolateElement, useState } from '@wordpress/element';
-import { Button, Notice } from '@wordpress/components';
+import { Button, Card, CardBody, Notice } from '@wordpress/components';
 import { PLUGINS_STORE_NAME } from '@woocommerce/data';
 import { useDispatch } from '@wordpress/data';
 import { recordEvent } from '@woocommerce/tracks';
 
 /**
  * Internal dependencies
  */
-import unconnectedImage from './subscriptions-empty-state-unconnected.svg';
+import Visa from './cards/visa.js';
+import MasterCard from './cards/mastercard.js';
+import Amex from './cards/amex.js';
+import DinersClub from './cards/diners.js';
+import Discover from './cards/discover.js';
+import JCB from './cards/jcb.js';
+import UnionPay from './cards/unionpay.js';
 import './style.scss';
```

Comment:
These card SVG components are defined elsewhere in repo, relative to this file: [`../payments-welcome/cards/`](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce-admin/client/payments-welcome/cards)

Do you think it's best to separate these files from the `payments-welcome` feature or share the existing files?

Reply:
Yeah, I noticed these. I decided to double up on these images because the `payments-welcome` is another experiment and I didn't want there to be any unintended side effects if/when they decide to end the experiment and remove the related code.